client/account-recovery/components/account-recovery-error-message/index.jsx

```diff
@@ -16,6 +16,12 @@ class AccountRecoveryErrorMessage extends Component {
 			case 'RestInvalidKeyError':
 				return translate( "We've failed to validate with the given code. " +
 					'Please double check if the code is correct.' );
+
+			// The error messages from our backend are quite descriptive in this case. e.g.
+			// "You've recently used this password", "Password must be at least 6 characeters.", etc.
+			// Thus I'm using it directly here.
+			case 'RestBadPasswordError':
+				return error.message;
```

Comment:
No, I was wrong. The response string is not localized :(
![image](https://cloud.githubusercontent.com/assets/1842898/25978742/bb353442-36f6-11e7-8d00-45fc91094880.png)

figuring out how to deal with this ...